collect-settings.rst

```diff
@@ -175,13 +175,182 @@ Admin settings allow you to :ref:`restrict which General Settings are seen by us
+When you click on the :guilabel:`Import/Export Settings`, you see a QR Code and a few options. 
 
-You can also import settings from a QR code saved to the device, by selecting :guilabel:`Select Code from SD Card`
+.. image:: /img/collect-settings/import-settings.*
+  :alt: Import/export settings menu of ODK Collect
+  :class: device-screen-vertical
 
-.. warning:: 
+QR Code
+""""""""
+
+QR Code or the Quick Response Code is a two dimensional barcode. QR codes can be used to configure Collect on many devices. The QR image presented contains all of your current General and Admin settings, including admin and server passwords. When this QR code is scanned from the ODK Collect app on another device, these settings are imported.
+
+Sharing QR code
+""""""""""""""""
+
+You can click on the |share| icon to share the generated barcode as an image. When you click on it, it displays a list of applications like *whatsapp*, *facebook* , *hangouts* to name a few, which can be used to share the generated QR code. This is useful when there are several different data collection sites and all devices have to be configured in the same way, in which case the QR code can be shared from one reference device. 
+
+.. |share| image:: /img/collect-settings/share-icon.*
+             :alt: Share icon for sharing the QR code. 
+             :height: 43 px
+             :width: 43 px
+
+Saving QR code locally
+""""""""""""""""""""""""
+
+You can go to :menuselection:`⋮ --> Save settings to disk` to save the QR code.  
+
+Importing settings from a QR saved on your device
+"""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+You can import settings from a QR code saved on your device by clicking on :guilabel:`Select Code from SD Card` option.
+
+Making your own QR code
+""""""""""""""""""""""""""
+
+To create your own QR code, you need to know that QR code is a JSON object with a structure as shown below:
+
+.. code-block:: JSON
+
+  {
+    "general": {
+      "protocol": "google_sheets",
+      "constraint_behavior": "on_finalize"
+    },
+    "admin": {
+      "edit_saved": false
+    }
+  }
+
+The JSON object isn't encrypted but is compressed using `zlib <http://www.zlib.net/manual.html>`_ before encoding into QRCode. Therefore the creation process can be summarized as follows:
+
+1) Write a JSON object containing the changed settings in form of structure as written above. 
+2) Compress it using zlib.
+3) Encode into QR code. 
+
+After you finish generating the QR code, you can transfer it to your device and then import it by clicking on :guilabel:`Select Code from SD Card`  option.
+
+Following is the list of all keys and the set of values they can take:
```

Comment:
This needs a section headline. Someone would want to link to the list as a reference item.